acceptancetests/assess_juju_output.py

```diff
@@ -106,7 +108,6 @@ def assess_juju_status(client, series):
     """
     deploy_charm_with_subordinate_charm(client, series)
     status = client.get_status()
-    verify_juju_status_fields(status)
```

Comment:
I'm guessing this actually solves a test?

Reply:
Not really sure what it does but given that:
a) it's missing
b) the bits below check specific status fields anyway

it seemed like the best approach would be to get rid of it.